seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/flow/TransformFlowLifeCycle.java

```diff
@@ -93,4 +102,12 @@ public void received(Record<?> record) {
     public void restoreState(List<ActionSubtaskState> actionStateList) throws Exception {
         // nothing
     }
+
+    @Override
+    public void close() throws IOException {
+        for (SeaTunnelTransform<T> t : transform) {
+            t.close();
```

Comment:
Hi, please catch close method on each transfrom, so we can make sure every transform close method can be called.

Reply:
ok